pkg/types/hashedrekord/v0.0.1/entry.go

```diff
@@ -230,10 +238,11 @@ func (v V001Entry) CreateFromArtifactProperties(_ context.Context, props types.A
 		return nil, errors.New("only one public key must be provided")
 	}
 
+	hashAlgorithm, hashValue := util.UnprefixSHA(props.ArtifactHash)
```

Comment:
this is to allow passing as `props.ArtifactHash` either a regular sha value, or a prefixed sha value, which can be used to determine what is the hash algorithm used.

Is this a breaking change? cc @haydentherapper